### PR TITLE
Credential Offer add subject, issuer validity checks

### PR DESCRIPTION
**Pull request by clauxx**

### CODE REVIEW

File: src/lib/interactionManager/credentialOfferFlow.ts [hunk 1]
```diff
@@ -77,13 +78,26 @@ export class CredentialOfferFlow extends Flow<
...
+
     this.state.credentialsAllValid = validArr.every(v => v)
     return true
   }
 
+  private async validateCredentials(signedCredentials: SignedCredential[]) {
+    return Promise.all(
+      signedCredentials.map(async cred => {
+        const validDigestable = await JolocomLib.util.validateDigestable(cred)
+        const validIssuer = cred.issuer !== this.ctx.participants.requester.did
```
Comment (mnzaki) [inline src/lib/interactionManager/credentialOfferFlow.ts:93]: Ummm no self-signed credentials??

File: src/lib/interactionManager/credentialOfferFlow.ts [hunk 2]
```diff
@@ -77,13 +78,26 @@ export class CredentialOfferFlow extends Flow<
...
+        const validSubject =
+          cred.subject !== this.ctx.participants.responder!.did
```
Comment (mnzaki) [inline src/lib/interactionManager/credentialOfferFlow.ts:95]: self-signed creds?

File: src/lib/interactionManager/credentialOfferFlow.ts (hunk 1 above)
Comment (clauxx) [reply src/lib/interactionManager/credentialOfferFlow.ts:93]: Should you be able to receive self-signed credentials from a different identity?

**Comment by mnzaki** [comment]
Any updates on this?

**Comment by Exulansis** [comment]
Can this be closed?

**Pull request closed without merging**